joint_state_controller/include/joint_state_controller/joint_command_controller.h

```diff
@@ -0,0 +1,75 @@
+/**
+Software License Agreement (proprietary)
+
+\file      joint_command_controller.h
+\authors   Peiyi Chen <pchen@clearpath.ai>
+\copyright Copyright (c) 2022, Clearpath Robotics, Inc., All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification, is not permitted without the
+express permission of Clearpath Robotics.
```

Comment:
I suggest we follow the license of the other packages in this repo? This license here means that if someone looks at this code in a browser it would be a license violation. (Also for the associated .cpp file)

Reply:
I swapped out the license for the clearpath BSD one which is in line with the other files. I don't think we can copy/paste the other license because it's for another company (hiDOF inc).